<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectAdd</class>
 <widget class="QWidget" name="ProjectAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>增加项目</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="1">
    <widget class="QTextEdit" name="desceribe_TE"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name_LE"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>项目名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>项目描述</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="add_PB">
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
